### PR TITLE
Enhance UI in Minting Details

### PR DESCRIPTION
**Pull request by ehab-hassan**
### Description

Enhance UI in Minting Details

### Changes

Enhanced the UI

image
image

List of changes this PR includes

### Related Issues

#2415

List of related issues

### Documentation PR

For UI changes, Please provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by MohamedElmdary — APPROVED** [review]

File: packages/playground/src/dashboard/components/NodeMintingDetails.vue [hunk 1]
```diff
@@ -1,65 +1,94 @@
...
+                  <v-list class="custom-list" density="compact">
+                    <h3 class="mb-1 font-weight-medium">
+                      Minting Details<v-chip class="ma-2 me-auto" small :color="getChipColor(receipt.type)"
+                        >{{ receipt.type }}
+                      </v-chip>
+                    </h3>
+                    <div class="border">
+                      <v-row class="row-style">
+                        <v-col class="py-1" cols="1" sm="2" style="min-width: fit-content">
+                          <v-list-item> Date of Payout :</v-list-item>
```
Comment (amiraabouhadid) [inline packages/playground/src/dashboard/components/NodeMintingDetails.vue:38]: looks good just please make this justify-content end to be consistent with other fields

File: packages/playground/src/dashboard/components/NodeMintingDetails.vue [hunk 2]
```diff
@@ -1,65 +1,94 @@
...
+                              {{ receipt.tft || 0 }} TFT<span v-if="receipt.fixupReward"
+                                >+ {{ receipt.fixupReward || 0 }} TFT FixedUp</span
+                              >
+                            </div>
+                          </v-list-item>
+                        </v-col>
+                      </v-row>
+                      <v-row v-for="(val, key) in receipt.cloud_units" :key="key" class="row-style">
+                        <v-col class="py-1" cols="1" sm="2" style="min-width: fit-content">
+                          <v-list-item> {{ key }} :</v-list-item>
```
Comment (amiraabouhadid) [inline packages/playground/src/dashboard/components/NodeMintingDetails.vue:64]: please capitalize CU, SU and NU and add their units. might be helpful to also add a tooltip explaining what they are

**Review by amiraabouhadid — CHANGES_REQUESTED** [review]

File: packages/playground/src/dashboard/components/NodeMintingDetails.vue (hunk 2 above)
Comment (ehab-hassan) [reply packages/playground/src/dashboard/components/NodeMintingDetails.vue]: done I added unit to SU & NU 
for CU there is no clear until but I think the tooltip explains
https://manual.grid.tf/knowledge_base/cloud/cloudunits.html?highlight=Cloud%20Units#cloud-units

**Comment by Mahmoud-Emad** [comment]
@amiraabouhadid Please re-review the added changes

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**